frontend/src/Components/LandingPage/Landingpage.js

```diff
@@ -17,8 +20,8 @@ function LandingPage() {
                 <p>New to our website? Join Now.</p>
                 <br/>
                 <div className="buttons">
-                    <button>Sign in</button>
-                    <button>Sign up</button>
+                    <button onClick={onSignIn}>Sign in</button>
+                    <button onClick={onSignUp}>Sign up</button>
```

Comment:
I think you can further improve the look of these buttons i.e coloring, shape etc because black text with grey background is a little hard to read.

Reply:
i have improved it